cips/cip-tx-interface.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Standardised Gas and Pricing Estimation Interface
+description: A standardised interface for estimating gas usage and gas pricing for transactions
+author: Callum Waters (@cmwaters)
+discussions-to: URL
```

Comment:
Note: this needs a URL pre-merge.

Reply:
Yeah I haven't yet created the forum post but will do that today